src/test/java/org/jabref/model/groups/AutomaticPersonsGroupTest.java

```diff
@@ -0,0 +1,51 @@
+package org.jabref.model.groups;
+
+import org.jabref.model.entry.BibEntry;
+import org.jabref.model.entry.field.StandardField;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+class AutomaticPersonsGroupTest {
+    @Test
+    void createSubgroupsFromCommaSeparatedLastNames() {
+        BibEntry bibEntry = new BibEntry().withField(StandardField.AUTHOR, "Turing, Alan and Hopper, Grace");
+        var subgroups = new AutomaticPersonsGroup("", GroupHierarchyType.INDEPENDENT, StandardField.AUTHOR).createSubgroups(bibEntry);
+        assertEquals(2, subgroups.size());
```

Comment:
It's better to construct a list of expected elements by hand and compare to that. For example, if the code happens to return 3 elements, you now only get a message saying "expected 2 but got 3" - not very helpful. But if you do
`expected = Arrays.asList(new Keywordgroup(...), ...); assertEquals(expected, subgroups)` you get a nice message, showing you which additional group is there.